src/Nox.Types.EntityFramework/Types/CountryCode3/CountryCode3DatabaseConfigurator.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Nox.Solution;
+using Nox.Types.EntityFramework.Abstractions;
+
+namespace Nox.Types.EntityFramework.Types;
+
+public class CountryCode3DatabaseConfigurator : INoxTypeDatabaseConfigurator
+{
+    public NoxType ForNoxType => NoxType.CountryCode3;
+    public bool IsDefault  => true;
+
+    public void ConfigureEntityProperty(
+        NoxSolutionCodeGeneratorState noxSolutionCodeGeneratorState,
+        EntityTypeBuilder builder,
+        NoxSimpleTypeDefinition property,
+        Entity entity,
+        bool isKey)
+    {
+        builder
+            .Property(property.Name)
+            .IsRequired(isKey || property.IsRequired)
```

Comment:
We don't need to check "isKey" anymore, based on discussion with @rochar, so please use only "property.IsRequired"